api/policies/v1/admissionpolicygroup_webhook_test.go

```diff
@@ -17,86 +15,39 @@ limitations under the License.
 package v1
 
 import (
+	"context"
 	"testing"
 
+	"github.com/go-logr/logr"
+	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+
 	admissionregistrationv1 "k8s.io/api/admissionregistration/v1"
 
 	"github.com/kubewarden/kubewarden-controller/internal/constants"
 )
 
 func TestAdmissionPolicyGroupDefault(t *testing.T) {
-	policy := AdmissionPolicyGroup{}
-	policy.Default()
+	defaulter := admissionPolicyGroupDefaulter{logger: logr.Discard()}
+	policy := &AdmissionPolicyGroup{}
 
-	require.Equal(t, constants.DefaultPolicyServer, policy.GetPolicyServer())
-	require.Contains(t, policy.GetFinalizers(), constants.KubewardenFinalizer)
-}
-
-func TestAdmissionPolicyGroupValidateCreate(t *testing.T) {
-	policy := NewAdmissionPolicyGroupFactory().Build()
-	warnings, err := policy.ValidateCreate()
+	err := defaulter.Default(context.Background(), policy)
 	require.NoError(t, err)
-	require.Empty(t, warnings)
-}
 
-func TestClusterAdmissionPolicyValidateCreateWithNoMembers(t *testing.T) {
```

Comment:
This and similar tests in other files are unnecessary since we already have unit tests for the creation without members: 
https://github.com/kubewarden/kubewarden-controller/blob/08d5afc37fb530089e817f20117810bce8f1e5da/api/policies/v1/policygroup_validation_test.go#L124


We just need a high-level test with the creation and the update errors to check that the wire-up of the validators was done correctly, so we can safely remove this.